packages/functional-tests/tests/syncV3/syncV3EmailFirst.spec.ts

```diff
@@ -31,6 +31,7 @@ test.describe('Firefox Desktop Sync v3 email first', () => {
 
     //Refresh the page
     await page.reload({ waitUntil: 'load' });
+    await page.waitForTimeout(1000);
```

Comment:
What's this for? Seems unrelated to legal. But wasnt sure if it's fixing some intermittent timing issues or debugging.